hashring/hashring.go

```diff
@@ -28,7 +28,9 @@ import (
 	"strings"
 	"sync"
 
+	log "github.com/uber-common/bark"
```

Comment:
Last PR we started reverting from doing this. `bark` should just be called `bark`, especially now that there is a new logging entity (the `logger` package).